dockerfiles/ci/centos/6/docker-compose.yml

```diff
@@ -5,73 +5,75 @@ services:
     image: 'datadog/dd-trace-ci:centos-6'
```

Comment:
I like the confidence :-) 
Levi mentioned he wanted to keep both alive for now (and he usually really likes to nuke things), should we consider removing everything now ?

Reply:
Just chiming in to confirm that I think nuking it just now is a bit premature. The more we nuke the harder it will be to restore in case we need to ^_^
Although I suppose it's easily copyable from the php-5 branch anyway?